### DISCUSSION TITLE
LazyTerm Toggle

### DISCUSSION THREAD

**Discussion by kobzar** [LazyVim/LazyVim#2621] [Q&A]
This is maybe stupid question but i can't find the work method to remap internal terminal to another key without Ctrl + combination.

By default we are has this one top Open/Hide terminal
``` lua
map("n", "<c-/>", lazyterm, { desc = "Terminal (root dir)" })
map("n", "<c-_>", lazyterm, { desc = "Open terminal" })
```
And both of them working fine.
But i am want the same actions by one key like "§", "F12", "±"(Shift+§) e.t.c, so i'll try to bind the same actions like 
``` lua
map("n", "§", lazyterm, { desc = "Terminal (root dir)" })
map("i", "§", lazyterm, { desc = "Open terminal" })
map("t", "§", lazyterm, { desc = "Open terminal" })
```
This is open the terminal!
But on next time when i press "§" key (terminal is opened and active in insert mode) the terminal isn't hided.
The new copy of terminal is opened instead hide/close.

So, my question hot to make toggle alternative for one key without "<c-§>"

**Comment by kobzar** [comment]
any updates?

**Comment by quiyahui** [comment]
I think what you want to do is map `§` to `:close<cr>`:

```lua
map("n", "§", lazyterm, { desc = "Open Terminal" })
map("i", "§", lazyterm, { desc = "Open Terminal" })
map("t", "§", "<cmd>close<cr>", { desc = "Hide Terminal" })
```

**Comment by kobzar** [comment]
Very very thanks! This is it!!!
